packages/react/src/CircleBadge/CircleBadge.stories.tsx

```diff
@@ -3,20 +3,21 @@ import type {Meta, StoryFn} from '@storybook/react'
 import CircleBadge from './CircleBadge'
 import {ZapIcon} from '@primer/octicons-react'
 
-export default {
+const meta: Meta<typeof CircleBadge> = {
   title: 'Components/CircleBadge',
   component: CircleBadge,
-} as Meta<typeof CircleBadge>
+}
+export default meta
 
 export const Default = () => (
```

Comment:
Could we add an `aria-label` to the default stories for `CircleBadge` and `CircleOcticon` (e.g. "User badge")?

Reply:
Sure thing